webodf/lib/ops/OpApplyHyperlink.js

```diff
@@ -101,7 +101,8 @@ ops.OpApplyHyperlink = function OpApplyHyperlink() {
             linkNode = createHyperlink(ownerDocument, hyperlink);
             node.parentNode.insertBefore(linkNode, node);
             linkNode.appendChild(node);
-            if (modifiedParagraphs.indexOf(paragraph) === -1) {
+            if (paragraph !== null
```

Comment:
getParagraphElement can take a ?Node. If that's null, it's passed straight through. So I prefer to keep the signature of getParagraphElement as it is.